GenderPayGap.WebUI/Views/Viewing/Launchpad/Parts/SearchByOrganisation.cshtml

```diff
@@ -0,0 +1,14 @@
+﻿@using (Html.BeginForm("SearchResults", "Viewing", FormMethod.Get, new {id = "FinderForm"}))
+{
+    <div class="form-group">
+        <div class="gov-uk-c-searchbar">
+            <div class="gov-uk-l-searchbar__table">
+                <label class="govuk-visually-hidden" for="organisation-search">Search by organisation</label>
+                <div class="gov-uk-l-searchbar__cell autocomplete-wrapper" id="organisation-search"></div>
```

Comment:
It might be worth a comment inside this `<div>` saying that it's going to be populated with the search bar - perhaps even pointing to the file that populates it.

Reply:
Good idea, I've added in a comment